study.md

```diff
@@ -100,7 +107,11 @@ pairs.
 | occupation | 'propane and propane accessories salesman' |
 
 ```javascript
-// your answer here
+let hank = {
+  family_name: 'hill',
+  given-name: 'hank',
```

Comment:
There needs to be quotation marks since there are special characters in the key name.  `'given-name': 'hank',`